tasks/demo/user/openPack.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script opens a pack
+
+configPath="../../.."
+signerBob=bob-account
+bobAddress=0x179b6b1cb6755e31
+
+cd $configPath
+
+# Get packs ids on Bob's Collection
+packsIDs=$(flow scripts execute ./scripts/mfl/packs/get_ids_in_collection.script.cdc $bobAddress | grep -o '\[.*\]')
+echo $packsIDs
```

Comment:
Can be nice to display a line before. "Packs ids in your wallet:". And maybe display "None" after that if there are none or the list of ids if there are some